test/service/test_response.py

```diff
@@ -23,6 +23,7 @@ class TestResponse(WebServiceTestCase):
 
     maxDiff = None
     bundles = WebServiceTestCase.bundles + [('fa5be5eb-2d64-49f5-8ed8-bd627ac9bc7a', '2019-02-14T192438.034764Z')]
+    bundles = bundles + [('d0e17014-9a58-4763-9e66-59894efbdaa8', '2018-10-03T144137.044509Z')]
```

Comment:
Just add the tuple to the list literal in the line above. With proper wrapping, of course.

And you know this: `x = x + y` is the same as `x += y`.